WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderUtilsWrapper.kt

```diff
@@ -41,4 +41,8 @@ class ReaderUtilsWrapper @Inject constructor(
     fun isExternalFeed(blogId: Long, feedId: Long): Boolean = ReaderUtils.isExternalFeed(blogId, feedId)
 
     fun getReportPostUrl(blogUrl: String): String = ReaderUtils.getReportPostUrl(blogUrl)
+
+    fun postAndCommentExists(blogId: Long, postId: Long, commentId: Long): Boolean {
```

Comment:
Wonder if it makes sense to add `is` to the beginning  of the name here, since we return a Boolean?

Reply:
That is a good point 😄  ! I thought to keep the `postAndCommentExists` name since we are wrapping an already existing function. I would have no doubt in changing this if it was a new piece of code, but maybe I would stay with the original wrapped name here? Wdyt?